### PR TITLE
add: `glow-deb`

### PR DESCRIPTION
**Pull request by mirenradia**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

File: packages/glow-deb/glow-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name="glow-deb"
+gives="glow"
+version="1.4.1"
+url="https://github.com/charmbracelet/${gives}/releases/download/v${version}/${gives}_${version}_linux_amd64.deb"
+description="Render markdown on the CLI, with pizzazz!"
+hash="1a57e45edeb0bfb78225a498c9764d8dc6796487a823098da4018f2fd9ad69df"
+maintainer="mirenradia <miren_radia@yahoo.co.uk>"
+repology=("project: glow")
+
```
Comment (Henryws) [inline packages/glow-deb/glow-deb.pacscript:9]: Remove this line

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/glow-deb/glow-deb.pacscript (hunk 1 above)
Comment (mirenradia) [reply packages/glow-deb/glow-deb.pacscript]: Done!

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `breaks`

**Comment by mirenradia** [comment]
> Add `breaks`

Can I add `replace` instead? I don't think this "breaks" any other pacstall package.

**Comment by Henryws** [comment]
Sure

File: packages/glow-deb/glow-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name="glow-deb"
+gives="glow"
+replace=${gives}
```
Comment (Henryws) [inline packages/glow-deb/glow-deb.pacscript:3]: Quote this

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/glow-deb/glow-deb.pacscript (hunk 2 above)
Comment (mirenradia) [reply packages/glow-deb/glow-deb.pacscript]: Ah yes. I missed that. Now fixed.

**Review by Henryws — APPROVED** [review]

**Pull request merged**